pkg/apis/serving/v1alpha2/openapi_generated.go

```diff
@@ -50,10 +50,6 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 		"./pkg/apis/serving/v1alpha2.TransformerSpec":         schema_pkg_apis_serving_v1alpha2_TransformerSpec(ref),
 		"./pkg/apis/serving/v1alpha2.TritonSpec":              schema_pkg_apis_serving_v1alpha2_TritonSpec(ref),
 		"./pkg/apis/serving/v1alpha2.XGBoostSpec":             schema_pkg_apis_serving_v1alpha2_XGBoostSpec(ref),
-		"knative.dev/pkg/apis.URL":                            schema_knativedev_pkg_apis_URL(ref),
```

Comment:
Any idea why this is deleted? I think we are still using them.

Reply:
I guess it's the generation. I didn't take it out manually. If I put it in manually it still seems to compile. My best guess right now is it's a knock-on from some upstream change in knative that's coming out of upgrading the version. Interesting that kservices are being created and their URLs do show in the status section even without these lines.

Reply:
ye this is to generate the openapi schema for KFServing python SDK. kubectl would still work but the python SDK client may not show the urls. I guess there is something wrong when generating knative types. We can revert this for now and create an issue to investigate the knative type generation issue.

Reply:
Have manually overridden in the code for now with a note and can create an issue to follow up.